microsoftgraph/microsoft-graph-explorer-v2
# integrates to samples api

## Overview

Fetches the samples from an API instead of a local JSON file

Fixes #147 

## Notes

In the rare case that a request to get the samples fails, we load a local limited set of samples. A message is also provided to the user.

![image](https://user-images.githubusercontent.com/53429535/64178425-21c62600-ce6a-11e9-8752-b532266e0ca5.png)


## Testing Instructions

* Load the app
* Notice a request is running on the sidebar
* Sample requests are displayed
